tests/sentry/integrations/vercel/test_webhook.py

```diff
@@ -259,3 +261,22 @@ def request_callback(request):
             assert len(responses.calls) == 2
             assert response.status_code == 400
             assert "Error setting refs" in response.data["detail"]
+
+    @responses.activate
+    def test_no_target(self):
```

Comment:
I feel like this should be named `test_manual_vercel_deploy` or something a little more clear